src/visualization/recipes_makie.jl

```diff
@@ -62,7 +65,7 @@ function iplot(u, mesh::UnstructuredMesh2D, equations, solver, cache;
   solution_z = Makie.@lift(getindex.($plotting_mesh.position, 3))
 
   # Plot the actual solution.
-  Makie.mesh!(ax, plotting_mesh, color=solution_z, nvisnodes=nvisnodes)
+  Makie.mesh!(ax, plotting_mesh, color=solution_z, nvisnodes=nvisnodes, colormap=colormap)
```

Comment:
```suggestion
  Makie.mesh!(ax, plotting_mesh; color=solution_z, nvisnodes, colormap)
```
No need for duplication if the names match.